etc/env/configs/via.toml

```diff
@@ -0,0 +1,63 @@
+__imports__ = [ "base", "l2-inits/dev.init.env" ]
+
+[via_btc_watch]
+confirmations_for_btc_msg=3
+btc_node_poll_interval=300
```

Comment:
This is the time required to restart an execution cycle in btc_watch tasks. The value is in milliseconds, let's set a bigger value.

Reply:
Let's make it to 1 sec. As we have new blocks each 5 seconds or so, it should be fine
but good catch